### DISCUSSION TITLE
语言问题

### DISCUSSION THREAD

**Discussion by h3h3da** [ctf-wiki/ctf-wiki#729] [Q&A] [answered]
更新之后的是不是换成英语会有问题？
![image](https://user-images.githubusercontent.com/20582659/104009262-bb7faf00-51e5-11eb-841b-e8b953c03eaf.png)

**Comment by iromise** [comment, marked as answer]
是的，文字也说明了具体的原因。原先的英语版本是用 Google Translate 翻译的，所以就去掉了。如果想要看英语（或者其它语言）版本的话，可以使用 Chrome 自带的页面翻译功能翻译，但这样的结果可能会不太可读。
